xml/System.ComponentModel/TypeDescriptionProvider.xml

```diff
@@ -437,9 +437,9 @@
         <Parameter Name="instance" Type="System.Object" Index="0" FrameworkAlternate="net-9.0" />
       </Parameters>
       <Docs>
-        <param name="instance">To be added.</param>
+        <param name="instance">The object for which to get the extended type descriptor.</param>
         <summary>Returns an extended custom type descriptor for the given object.</summary>
-        <returns>To be added.</returns>
+        <returns>An <see cref="T:System.ComponentModel.ICustomTypeDescriptor" /> that can provide extended metadata for the object.</returns>
         <remarks>An extended type descriptor is a custom type descriptor that offers properties that other objects have added to this object, but are not actually defined on the object. For example, in the .NET Framework Component Model, objects that implement the interface IExtenderProvider can "attach" properties to other objects that reside in the same logical container. The GetTypeDescriptor method does not return a type descriptor that provides these extra extended properties. GetExtendedTypeDescriptor returns the set of these extended properties. TypeDescriptor will automatically merge the results of these two property collections. Note that while the .NET Framework component model only supports extended properties this API can be used for extended attributes and events as well, if the type description provider supports it.</remarks>
```

Comment:
Typo from original .cs file; this:
```GetExtendedTypeDescriptor returns the set of these extended properties.```
should be
```GetExtendedTypeDescriptorFromRegisteredType returns the set of these extended properties.```
or perhaps just replace with "This method returns..." as is done elsewhere in this area.

Reply:
```suggestion
        <remarks>An extended type descriptor is a custom type descriptor that offers properties that other objects have added to this object, but are not actually defined on the object. For example, in the .NET Framework Component Model, objects that implement the interface IExtenderProvider can "attach" properties to other objects that reside in the same logical container. The GetTypeDescriptor method does not return a type descriptor that provides these extra extended properties. GetExtendedTypeDescriptor returns the set of these extended properties. TypeDescriptor will automatically merge the results of these two property collections. Note that while the .NET Framework component model only supports extended properties this API can be used for extended attributes and events as well, if the type description provider supports it.</remarks>
```
```suggestion
        <remarks>An extended type descriptor is a custom type descriptor that offers properties that other objects have added to this object, but are not actually defined on the object. For example, in the .NET Framework Component Model, objects that implement the interface IExtenderProvider can "attach" properties to other objects that reside in the same logical container. The GetTypeDescriptorFromRegisteredType method does not return a type descriptor that provides these extra extended properties. GetExtendedTypeDescriptorFromRegisteredType returns the set of these extended properties. TypeDescriptor will automatically merge the results of these two property collections. Note that while the .NET Framework component model only supports extended properties this API can be used for extended attributes and events as well, if the type description provider supports it.</remarks>
```